app_dart/lib/src/request_handlers/check_for_waiting_pull_requests_queries.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Just curious, how did you debug the GraphQL in this file?

Reply:
https://developer.github.com/v4/explorer/

I also had a local dart program that was basically the `get` method o fthe request handler